recipes/scikit-cycling/meta.yaml

```diff
@@ -48,7 +48,7 @@ test:
   imports:
     - skcycling
   commands:
-    - pytest skcycling --exe
+    - pytest skcycling
```

Comment:
You may need to copy the pytest folder from source if you are going that route. Check this https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#source-files

Reply:
I forgot `--pyargs` which should find the location where the installed version is and collect the tests.